app/assets/stylesheets/nfg_ui/network_for_good/admin/nfg_theme/custom/_sys_only_functionality.scss

```diff
@@ -0,0 +1,51 @@
+// Heartbeat "beacon" styles for sysuser functionality
+.notify {
+  position: absolute;
+  top: 0;
+  right: 0;
+  z-index: 50;
+  .heartbit {
+    position: absolute;
+    top: -($spacer * .75);
+    right: -($spacer * .75);
+    height: ($spacer * 1.5);
+    width: ($spacer * 1.5);
+    z-index: 10;
+    border: 6px solid $warning;
+    border-radius: ($spacer * .75);
+    animation: heartbit 1s ease-out infinite;
+  }
+  .point {
+    position: absolute;
+    right: -($spacer * .25);
+    top: -($spacer * .25);
+    width: ($spacer * .5);
+    height: ($spacer * .5);
+    border-radius: $spacer;
+    background-color: $warning;
+  }
+}
+
+// Heartbeat animation used on beacon for sysusers
+@keyframes heartbit {
```

Comment:
@kylejmorin is `heartbit` the correct spelling?

Reply:
I questioned that to when bringing these styles over. I assume this was pulled from some CSS demo this functionality was built from. I think it's also part of the JS in DMS.